test/util/testnode/sign.go

```diff
@@ -20,7 +20,13 @@ func SignAndBroadcastTx(encCfg encoding.Config, c client.Context, account string
 			sdk.NewCoin(app.BondDenom, sdk.NewInt(1)),
 		)),
 	}
+	return SignAndBroadcastTxWithBuilderOption(opts, encCfg, c, account, msg...)
+}
 
+// SignAndBroadcastTxWithBuilderOption signs a transaction using the provided account and keyring
+// it accepts customized Tx builder options
+// inside the client.Context, then broadcasts it synchronously.
```

Comment:
[nit] rewrap + complete sentences

```suggestion
// SignAndBroadcastTxWithBuilderOption signs a transaction using the provided
// account and keyring. It accepts customized Tx builder options inside the
// client.Context, then broadcasts it synchronously.
```